gallery/transforms/plot_rotated_box_transforms.py

```diff
@@ -0,0 +1,108 @@
+"""
+===============================================================
+Transforms on Rotated Bounding Boxes
+===============================================================
+
+Introduction.
+
+First, some code to set everything up.
+"""
+
+# %%
+from PIL import Image
+from pathlib import Path
+import matplotlib.pyplot as plt
+
+
+import torch
+from torchvision import tv_tensors
+from torchvision.transforms import v2
+from helpers import plot
+
+plt.rcParams["figure.figsize"] = [10, 5]
+plt.rcParams["savefig.bbox"] = "tight"
+
+# if you change the seed, make sure that the randomly-applied transforms
+# properly show that the image can be both transformed and *not* transformed!
+torch.manual_seed(0)
+
+# If you're trying to run that on Colab, you can download the assets and the
+# helpers from https://github.com/pytorch/vision/tree/main/gallery/
+orig_img = Image.open(Path('../assets') / 'leaning_tower.jpg')
+
+# %%
+# Rotated Bounding Boxes
+# ----------------------
+# Brief intro into what rotated bounding boxes are. Brief description of the
+# image.
+
+orig_box = tv_tensors.BoundingBoxes(
+    [
+        [860.0, 1100, 570, 1840, -7],
+    ],
+    format="CXCYWHR",
+    canvas_size=(orig_img.size[1], orig_img.size[0]),
+    clamping_mode="hard",
+)
+# TODO: why is this necessary?
+orig_box = v2.ConvertBoundingBoxFormat("xyxyxyxy")(orig_box)
```

Comment:
:eyes:

Reply:
I figured out why, explained in code comments.